centos_ci.sh

```diff
@@ -208,27 +205,101 @@ function docs_tar_upload() {
   echo "Find docs tar here http://artifacts.ci.centos.org/minishift/minishift/docs/$LATEST."
 }
 
-if [[ "$UID" = 0 ]]; then
+function create_release_commit() {
+  # Set Terminal
+  export TERM=xterm-256color
+  # Add git a/c identity
+  git config user.email "minishift@lists.minishift.io"
```

Comment:
-1 for using minishift@lists.minishift.io. Github sends emails for confirmation  or when the bot is subscribed to an repository or when it is mentioned in a PR. It will really bad when these emails go to public mailing list.